src/components/SeverityIndicator/SeverityIndicator.tsx

```diff
@@ -2,23 +2,24 @@ import { Tooltip } from '@konturio/ui-kit';
 import { memo, useRef, useState } from 'react';
 import { i18n } from '~core/localization';
 import s from './SeverityIndicator.module.css';
+import type { Severity } from '~core/types';
 
 const COLORS = ['#FFDF35', '#FFB800', '#FF8A00', '#FF3D00', '#EA2A00'];
 
-const SEVERITY_TO_TEXT = {
-  UNKNOWN: i18n.t('common.severity_unknown'),
-  TERMINATION: i18n.t('common.severity_termination'),
-  MINOR: i18n.t('common.severity_minor'),
-  MODERATE: i18n.t('common.severity_moderate'),
-  SEVERE: i18n.t('common.severity_severe'),
-  EXTREME: i18n.t('common.severity_extreme'),
+const severityToText = (severity: Severity) => {
+  const translations = {
+    UNKNOWN: i18n.t('event_list.severity_unknown'),
+    TERMINATION: i18n.t('event_list.severity_termination'),
+    MINOR: i18n.t('event_list.severity_minor'),
+    MODERATE: i18n.t('event_list.severity_moderate'),
+    SEVERE: i18n.t('event_list.severity_severe'),
+    EXTREME: i18n.t('event_list.severity_extreme'),
+  };
+
+  return translations[severity];
 };
```

Comment:
just not to call all translations even if they not needed: 
```suggestion
const severityToText = (severity: Severity) => {
  switch (severity) {
    case 'UNKNOWN':
      return i18n.t('event_list.severity_unknown');
    case 'TERMINATION':
      return i18n.t('event_list.severity_termination');
    case 'MINOR':
      return i18n.t('event_list.severity_minor');
    case 'MODERATE':
      return i18n.t('event_list.severity_moderate');
    case 'SEVERE':
      return i18n.t('event_list.severity_severe');
    case 'EXTREME':
      return i18n.t('event_list.severity_extreme');
  }
};
```

Reply:
I thought about switch-case but decided that the object be a more compact construction